content/articles/The-concept-of-cookies-in-servlets/index.md

```diff
@@ -0,0 +1,222 @@
+- [Cookie class and the methods required for cookies](#cookie-class-and-the-methods-required-for-cookies)
+   - [Cookie class](#cookie-class)
+   - [Methods of Cookie class](#methods-of-cookie-class)
+- [How to create, delete, and get cookies](#how-to-create-delete-and-get-cookies)
+  - [Creating a new Cookie object](#creating-a-new-cookie-object)
+  - [Setting up lifespan for a cookie](#setting-up-lifespan-for-a-cookie)
+  - [Sending the cookies to the client](#sending-the-cookies-to-the-client)
+  - [Getting cookie from client request](#getting-cookie-from-client-request)
+  - [Deletting cookies](#deletting-cookies)
+- [Example to demonstrate Servlet Cookie](#example-to-demonstrate-servlet-cookie)
+- [Conclusion](#conclusion)
+
+### API for Cookies
+
+The `Cookie` class in the Servlet API is used to create cookies. Cookies are created using the Servlet API's 'Cookie' class. This class employs a variety of approaches to give various cookie activities. The `addCookie()` method, for example, is used to add cookies to the response object. It sends cookie data from a client to a server or from a server to a client using the `HTTP` response. The `getCookies()` method, on the other hand, is used to access the cookies that have been added to the response object.
+
+### How does Cookie work
+When a client makes a new request to a Web Server, the web server assigns a unique ID as a `cookie`, which is then sent back with the response to the client and saved in the client's browser.
+
+This `cookie` value is likewise provided to the server when the client sends another request, and the server uses this value to identify the user.
+
+In this case, a cookie is used to store information required each time the client visits a website, such as a user ID, name, or other identifiers.
+### Types of Cookies in Java Servlet
+Two types of cookies in the servlet are Non-persistent and persistent cookies.
+
+#### Non persistent cookies
+
+A non-persistent/Session cookie is only valid for one session and is automatically deleted when the browser is closed.
+
+#### persistent cookies
+
+It's valid for multiple sessions and, unlike non-persistent cookies, it's not deleted when the user closes the browser; instead, it's deleted only when the user logs out or signs out.
+
+### Advantage of Cookies
+
+- Transparent: Cookies work in this case without the user being aware that data is being saved in their browser.
+- Simplest technique: Cookies are the most basic method for keeping client states while connected to a web server.
+- Because cookies are stored in the client browser, they reduce the load on the server's memory.
+### Disadvantage 
+-  Cookies will only work if the user's web browser allows them; otherwise, they will not.
+- In a Cookie object, only textual data can be stored.
+
+### Cookie class and the methods required for cookies
+
+#### Cookie class
+ A `Cookie`, is a class that contains several cookie-related methods that allow you to use cookies.

```

Comment:
Change to " A `Cookie` class contains several cookie-related methods that allow you to use cookies."